### ISSUE TITLE
don't move to next tab

### ISSUE THREAD

**Issue by maticrivo** [chabou/hyper-pane#17]
when pressing `ctrl+alt+right` or `ctrl+alt+left` it also changes tabs which is not the required behavior. is it possible to change that to only cycle between current tab panes? even if its under a flag such as `cycleTabs: false` it can be great.

**Comment by chabou** [comment]
I can't reproduce.
What is your OS and your `hyper` config ?

**Comment by tjbenton** [comment]
I'm experiencing the same behavior with `ctrl+alt+up` or `ctrl+alt+down`.

```
macOS Sierra
version: 10.12.6
```
```
Hyper 1.4.3 (stable)
Plugins: 
  hyperterm-1password (2.0.1),
  hyperterm-tabs (1.3.5),
  hypertitle (1.0.1),
  hypercwd (1.1.1),
  hyperterm-material (0.1.8),
  hyperterm-summon (1.6.1),
  hyper-search (0.0.6),
  hyper-pane (1.3.0),
  hyperterm-tab-icons (1.1.0),
  hyper-color-command (1.0.0),
  hyper-font-smoothing (1.0.0),
  hyperemoji (0.0.4)
```

```js
// Hyper config
module.exports={
  config: {
    fontSize: 15,
    fontFamily: 'Source Code Pro',
    cursorColor: 'rgba(248,28,229,0.75)',
    cursorShape: 'BLOCK',
    foregroundColor: '#fff',
    backgroundColor: '#000',
    borderColor: '#333',
    css: '',
    termCSS: '',
    padding: '0px 5px 5px',
    colors: {
      black: '#000000',
      red: '#ff0000',
      green: '#33ff00',
      yellow: '#ffff00',
      blue: '#0066ff',
      magenta: '#cc00ff',
      cyan: '#00ffff',
      white: '#d0d0d0',
      lightBlack: '#808080',
      lightRed: '#ff0000',
      lightGreen: '#33ff00',
      lightYellow: '#ffff00',
      lightBlue: '#0066ff',
      lightMagenta: '#cc00ff',
      lightCyan: '#00ffff',
      lightWhite: '#ffffff'
    },
    shell: '',
    summonShortcut: 'Option+`',
  },
  plugins: [
    'hyperterm-1password',
    'hyperterm-tabs',
    "hypertitle",
    "hypercwd",
    "hyperterm-material",
    "hyperterm-summon",
    "hyper-search",
    "hyper-pane",
    "hyperterm-tab-icons",
    "hyper-color-command",
    "hyper-font-smoothing",
    "hyperemoji"
  ],
  localPlugins: []
}
```

**Comment by chabou** [comment]
@tjbenton I can't reproduce 😞  Even with your config.
Maybe you have some global shortcuts (other application or OS) that prevent Hyper to catch `ctrl+alt+up/down`.
Try to configure to other keys.

**Comment by chabou** [comment]
Oh I misunderstood this issue. Reproduced.
It returns to first tab when your are navigating to other pane in another tab.
Will fix this ASAP.

Thank you so much for your feedback 🙏

**Comment by chabou** [comment]
Fixed in release v1.3.1
